tests/__snapshots__/index.test.js.snap

```diff
@@ -1759,7 +1927,22 @@ exports[`createbom produces a BOM in JSON format that includes hashes from packa
       ]
     }
   },
-  \\"components\\": []
+  \\"components\\": [],
+  \\"dependencies\\": [
+    {
+      \\"ref\\": \\"pkg:npm/%40cyclonedx/bom-test-with-lockfile-2@1.1.3\\",
+      \\"dependsOn\\": [
```

Comment:
:question:  is it intended to store a list of `null` at this point?

Reply:
I will check with this and get back to you

Reply:
Thanks for catching this, I have fixed it. Please have a look now.